tests/bsim/bluetooth/audio/src/cap_acceptor_test.c

```diff
@@ -401,14 +401,23 @@ static int bis_sync_req_cb(struct bt_conn *conn,
 			   const struct bt_bap_scan_delegator_recv_state *recv_state,
 			   const uint32_t bis_sync_req[CONFIG_BT_BAP_BASS_MAX_SUBGROUPS])
 {
-	/* We only care about a single subgroup in this test */
+	uint32_t total_bis = 0U;
+
 	broadcaster_broadcast_id = recv_state->broadcast_id;
-	if (bis_sync_req[0] != 0) {
+
+	for (int i = 0; i < recv_state->num_subgroups; i++) {
+		total_bis |= bis_sync_req[i];
+	}
+
+	if (total_bis != 0U) {
 		SET_FLAG(flag_bis_sync_requested);
 	} else {
 		UNSET_FLAG(flag_bis_sync_requested);
 	}
 
+	/* total_bis is the new bis-sync setting to use when calling create_and_sync_sink */
+	bis_index_bitfield = total_bis;
```

Comment:
This will get overwritten by `base_recv_cb`. 

We should either use the bitfield from here, or from `base_recv_cb`, but using both like this will make it timing dependent on which is used

Reply:
I'll revert, so that we use the bitfield from base_recv_cb